packages/apollo-server-core/src/graphqlOptions.ts

```diff
@@ -58,7 +58,7 @@ export interface GraphQLServerOptions<
   plugins?: ApolloServerPlugin[];
   documentStore?: InMemoryLRUCache<DocumentNode>;
   parseOptions?: ParseOptions;
-  __testing_nodeEnv__?: string | undefined;
+  nodeEnv?: string | undefined;
```

Comment:
Think this should can actually be required, since we will be defaulting to `NODE_ENV` and can be certain that _something_ will always be there.